src/sass/styles.scss

```
@@ -259,3 +259,7 @@ $border-thin: 1px solid $color-mid-light !default;
 .no-border-top {
   border-top: none !important;
 }
+
+.text-area {
```

Also, maybe this can be a bit more specific, to track down the context where it was added:

```suggestion
.auto-expanding-textarea {
```

---

Sure, you could do that, or also keep it where it is - to avoid migrating all the other custom styles for the components that we might have in `styles.scss` at the moment.

By and large I think it's good to have it in `styles.scss` as it's just a custom style for a tiny custom component, and it would be an overkill to create a dedicated file for it - I think this is the reason why this file is now full of styles for different custom components. However, exactly due to the fact that we have a lot of classes in this file, we should try to be as specific as possible in terms of naming, so that just by the name one can grasp what is being done and why. :)

---

okay perfect, will keep it as is with specific naming then, I can get behind that logic :+1: thanks!